packages/browser/src/plugins/remote-loader/index.ts

```diff
@@ -86,6 +89,13 @@ export class ActionDestination implements DestinationPlugin {
           methodName,
           type: 'action',
         })
+
+        if (!(await this.ready())) {
+          throw new Error(
+            'Something prevented the destination from getting ready'
+          )
+        }
```

Comment:
I'd recommend moving this above the metric since we use the metric to understand how often destinations are called. If we introduced a regression here (where this.ready() never resolved), our metrics would look unchanged right now.

Reply:
Good catch! done